### DISCUSSION TITLE
Custom npm install command to include --force

### DISCUSSION THREAD

**Discussion by vasquez-dev** [renovatebot/renovate#25012] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.6.1

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

```
npm install --force --package-lock-only --no-audit --ignore-scripts <dependency>
```

### What you tried so far.

I've tried adding `postUpgradeTasks` to force install but the installation errors before the commit is made so it errors out.
We are looking to resolve the underlying issue requiring the force but for now we're looking for a solution to be able to configure renovate with --force

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: rawExec err (repository=redacted, branch=commit-and-tag-version-11.x-lockfile)
       "err": {
         "cmd": "/bin/sh -c npm install --package-lock-only --no-audit --ignore-scripts commit-and-tag-version@11.2.4",
         "stderr": "npm ERR! code ERESOLVE\nnpm ERR! ERESOLVE could not resolve\nnpm ERR! \nnpm ERR! While resolving: @casl/angular@7.1.0 
         ..
         @casl/angular@\"^7.0.0\" from the root project\nnpm ERR! \nnpm ERR! Fix the upstream dependency conflict, or retry\nnpm ERR! this command with --force, or --legacy-peer-deps\nnpm ERR! to accept an incorrect (and potentially broken) dependency resolution.\nnpm ERR!",
```

</details>

**Comment by vasquez-dev** [comment]
I saw another discussion suggesting using the `.npmrc` file. 
I tried that suggestion but it doesn't seem to be applying force on the npm install commands

**Comment by vasquez-dev** [comment, marked as answer]
Was able to get this going. 
I ended up setting 
```
  "npmrc": "force=true",
```
in the child repo.
Not sure why since the parent repo already had it configured in it's `.npmrc` file.

**Reply by jpgoelz** [reply]
Unfortunately, this did not work for me.
![image](https://github.com/renovatebot/renovate/assets/32487587/5444c030-a666-4b19-a511-d9b8c2c5b6fb)
